examples/example-kotlin/src/main/kotlin/io/grpc/examples/helloworld/HelloWorldClient.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2015, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.examples.helloworld
+
+import io.grpc.ManagedChannel
+import io.grpc.ManagedChannelBuilder
+import io.grpc.StatusRuntimeException
+import java.util.concurrent.TimeUnit
+import java.util.logging.Level
+import java.util.logging.Logger
+
+/**
+ * A simple client that requests a greeting from the [HelloWorldServer].
+ */
+class HelloWorldClient
+/** Construct client for accessing RouteGuide server using the existing channel.  */
+internal constructor(private val channel: ManagedChannel) {
+    private val blockingStub: GreeterGrpc.GreeterBlockingStub
+            = GreeterGrpc.newBlockingStub(channel);
+
+    /** Construct client connecting to HelloWorld server at `host:port`.  */
+    constructor(host: String, port: Int) : this(ManagedChannelBuilder.forAddress(host, port)
+            // Channels are secure by default (via SSL/TLS). For the example we disable TLS to avoid
+            // needing certificates.
+            .usePlaintext(true)
+            .build()) {
+    }
+
+
+    @Throws(InterruptedException::class)
+    fun shutdown() {
+        channel.shutdown().awaitTermination(5, TimeUnit.SECONDS)
+    }
+
+    /** Say hello to server.  */
+    fun greet(name: String) {
+        logger.log(Level.INFO, "Will try to greet {0}...", name)
+        val request = HelloRequest.newBuilder().setName(name).build()
+        val response: HelloReply =  try {
+            blockingStub.sayHello(request)
+        } catch (e: StatusRuntimeException) {
+            logger.log(Level.WARNING, "RPC failed: {0}", e.status)
+            return;
```

Comment:
multiple `;`s were introduced in later commits
otherwise LGTM for Kotlin dialect.

Reply:
Fixed